src/main/java/wolox/training/models/User.java

```diff
@@ -57,23 +79,23 @@ public String getName() {
 	}
 
 	public void setName(String name) {
-		this.name = name;
+		this.name = checkNotNull(name, PreconditionsConstants.NAME_CANT_BE_NULL);
```

Comment:
in the strings it is also good to validate that the text is not null
`Preconditions.checkArgument(!Strings.isNullOrEmpty(name), PreconditionsConstants.NAME_CANT_BE_NULL);`

Reply:
Done!